ej4/Chevybuilder.java

```diff
@@ -0,0 +1,7 @@
+
+public class Chevybuilder extends Chatabuilder {
```

Comment:
Remember to use UpperCamelCase to object names and lowerCamelCase to variable names [CamelCase](https://es.wikipedia.org/wiki/CamelCase)